models/public/detr-resnet50/README.md

```diff
@@ -0,0 +1,104 @@
+# detr-resnet50
+
+## Use Case and High-Level Description
+
+The `detr-resnet50` model is a model of the new framework, called DEtection TRansformer or DETR with ResNet50 backbone for object detection task. DETR predicts all objects at once, and is trained end-to-end with a set loss function which performs bipartite matching between predicted and ground-truth objects. DETR simplifies the detection pipeline by dropping multiple hand-designed components that encode prior knowledge, like spatial anchors or non-maximal suppression.
+
+More details provided in the [paper](https://arxiv.org/abs/2005.12872) and [repository](https://github.com/facebookresearch/detr).
+
+## Specification
+
+| Metric                          | Value             |
+|---------------------------------|-------------------|
+| Type                            | Object detection  |
+| GFLOPs                          | 174.4708          |
+| MParams                         | 41.3293           |
+| Source framework                | PyTorch\*         |
+
+## Accuracy
+
+Accuracy metrics obtained on [Common Objects in Context (COCO)](https://cocodataset.org/#home) validation dataset for converted model. Background label + label map with 80 public available object categories are used (original indexing to 91 categories is preserved). Object classes are listed in `<omz_dir>/data/dataset_classes/coco_91cl_bkgr.txt` file.
+
+| Metric              | Value  |
+| ------------------- | ------ |
+| coco_orig_precision | 39.27% |
+| coco_precision      | 42.36% |
+
+## Input
+
+### Original model
+
+Image, name - `input`, shape - `1, 3, 800, 1137`, format `B, C, H, W`, where:
+
+- `B` - batch size
+- `C` - number of channels
+- `H` - image height
+- `W` - image width
+
+Expected color order is `RGB`.
+
+Mean values - [123.675, 116.28, 103.53].
+Scale values - [58.395, 57.12, 57.375].
+
+### Converted model
+
+Image, name - `input`, shape - `1, 3, 800, 1137`, format `B, C, H, W`, where:
+
+- `B` - batch size
+- `C` - number of channels
+- `H` - image height
+- `W` - image width
+
+Expected color order is `BGR`.
+
+## Output
+
+### Original model
+
+1. Bounding boxes, name: `boxes`, shape - `1, 100, 4`. Presented in format `B, A, 4`, where:
+
+    - `B` - batch size
+    - `A` - number of detected anchors
+
+    For each detection, the description has the format: [`x`, `y`, `w`, `h`], where:
+
+    - (`x`, `y`) - coordinates of the bounding box center(coordinates are in normalized format, in range [0, 1])
+    - `w` - width of bounding box(values are in normalized format, in range [0, 1])
+    - `h` - height of bounding box(values are in normalized format, in range [0, 1])
+
+2. Scores, name: `scores`, shape - `1, 100, 92`. Contains scores for background and 91 object classes which are listed in `<omz_dir>/data/dataset_classes/coco_91cl_bkgr.txt` file.
```

Comment:
but in demo code you disable the last class, does it means that background (or no_detection in DETR terminology is the last class) and if you run demo with provided class names, names will be incorrect?